_posts/ericabrody/2016-02-22-ericabrody-drawing-clinic2.md

```diff
@@ -0,0 +1,29 @@
+ ---
```

Comment:
I believe this space here is making your drawing program not show up on this post.  I assume you and @payalpn used screenshare directly from the trinket but still both of you should've checked the post after merging it.  Please work together to fix this when you can